src/backend/tests/Fusion.Testing.Mocks.ProfileService/Fusion.Testing.Mocks.ProfileService.csproj

```diff
@@ -8,6 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Bogus" Version="33.0.2" />
     <PackageReference Include="Fusion.Integration.Profile.Abstractions" Version="5.3.0" />
+    <PackageReference Include="Fusion.ApiClients.Org" Version="5.0.6" />
```

Comment:
I would suggest to remove this dependency, and create test-models for required tests.